src/test/java/com/deque/axe/android/utils/AxeJankyPng.java

```diff
@@ -76,7 +77,8 @@ public AxeJankyPng(AxeFile axeFile) {
   }
 
   public AxeJankyPng(final String base64png) {
-    this(new PngReader(new ByteArrayInputStream(Base64.getDecoder().decode(base64png))));
+    this(new PngReader(new ByteArrayInputStream(Base64.getMimeDecoder()
```

Comment:
@ryomtoob didn't we need to change this at some point? I feel like we've gone back and forth on this line of code.

Reply:
Just talked to Rachel about this, we think on web the browser takes care of decoding the image. We never had to change the decoding logic because we did not update any json which have screenshots. In `api_test.json` I used the latest type of screenshot which has to be decoded using `mime decoder`. I tested this scenario by putting new screenshot in one of our old test spec jsons and used Base64 decoder instead of Base 64 mime decoder, it failed because of decoder.